Sources/NestKit/Assets/AssetRegistryClientBuilder.swift

```diff
@@ -11,13 +11,19 @@ public struct GitRepositoryClientBuilder: Sendable {
         self.authToken = authToken
         self.logger = logger
     }
-
-    public func build(for url: GitURL) -> any GitRepositoryClient {
+    
+    /// Build AssetRegistryClient based on the given git url.
+    ///
+    /// > Note: This function currently supports only GitHub.
+    public func build(for url: GitURL) -> any AssetRegistryClient {
         // Only GitHub is supported now.
         GitHubRepositoryClient(httpClient: httpClient, authToken: authToken, logger: logger)
```

Comment:
`GitHubRepositoryClient` is inherited from `AssetRegistryClient`. How about renaming this, too?